### PR TITLE
docs: fix links, add info box for rollback alarms

### PR DESCRIPTION
**Pull request by huanjani**
Explains that ABR deployment setting is ignored if associated alarms are already in alarm state. Also adds link to ECS docs. Fixes broken links.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/include/deployment.en.md [hunk 1]
```diff
@@ -9,8 +9,11 @@ Rolling deployment strategy. Valid values are
...
 - `"recreate"`: Stop all running tasks and then spin up new tasks. Under the hood, this translates to setting the [`minimumHealthyPercent`](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service_definition_parameters.html#minimumHealthyPercent) to 0 and [`maximumPercent`](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service_definition_parameters.html#maximumPercent) to 100.
 
-<span class="parent-field">deployment.</span><a id="deployment-rollback-alarms" href="#deployment-rollback-alarms" class="field">`rollback_alarms`</a> <span class="type">Array of Strings or Map</span>  
-As a list of strings, the names of existing CloudWatch alarms to associate with your service that may trigger a deployment rollback.
+<span class="parent-field">deployment.</span><a id="deployment-rollback-alarms" href="#deployment-rollback-alarms" class="field">`rollback_alarms`</a> <span class="type">Array of Strings or Map</span>
+!!! info
+    If an alarm is in the ALARM state at the beginning of a deployment, Amazon ECS will not monitor alarms for the duration of that deployment (Amazon ECS ignores the alarm configuration).
```
Comment (iamhopaul123) [inline site/content/docs/include/deployment.en.md]: Maybe?
```suggestion
    If an alarm is in the "ALARM" state at the beginning of a deployment, Amazon ECS will NOT monitor alarms for the duration of that deployment.
```

**Review by iamhopaul123 — APPROVED** [review]
🙏

File: site/content/docs/include/deployment.en.md (hunk 1 above)
Comment (iamhopaul123) [reply site/content/docs/include/deployment.en.md]: Also it will probably be better if we link official doc here.

Comment (huanjani) [reply site/content/docs/include/deployment.en.md]: Okay. I'll link in both places for better coverage. And I'll use the actual wording if we're putting it in quotes!

**Review by Lou1415926 — APPROVED** [review]

**Pull request merged**